src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days === 3) {
+    return 100;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 7) {
+    return 230;
+  }
+
+  return 80;
```

Comment:
The current implementation only returns specific values for 3, 6, and 7 days, and defaults to 80 for all other cases. This approach lacks flexibility and does not account for other possible day values. Consider implementing a more dynamic calculation method, such as a base rate with discounts for certain day ranges, to handle a wider range of inputs.